examples/deploy.py

```diff
@@ -14,15 +14,15 @@ def run() -> None:
     client = ExonumClient(hostname="127.0.0.1", public_api_port=8080, private_api_port=8081)
 
     # Name of the artifact:
-    service_name = "exonum-cryptocurrency-advanced:0.12.0"
+    service_name = "exonum-cryptocurrency-advanced:0.13.0-rc.2"
```

Comment:
Can we create a var with service name/version, so we don't have to change it in several places if name/version is changed?

Reply:
`service_name` is actually the required variable, but it is not fully used )